deploy/helm/kubecf/templates/ops-add-autoscaler.yaml

```diff
@@ -0,0 +1,1422 @@
+{{- if .Values.features.autoscaler.enabled -}}
```

Comment:
This could be in the file `deploy/helm/kubecf/assets/operations/instance_groups/app_autoscaler.yaml` (we use snake-case for the directory structure) like https://github.com/SUSE/kubecf/blob/master/deploy/helm/kubecf/assets/operations/instance_groups/eirini.yaml.

This way, all the operations are in pure YAML and we can eventually lint them (https://github.com/SUSE/kubecf/issues/23).

Reply:
Thanks for helping review this PR. I have moved the files to `deploy/helm/kubecf/assets/operations/instance_groups/app_autoscaler.yaml` as you suggested and manually test locally.